scaleway/resource_k8s_cluster.go

```diff
@@ -157,6 +158,13 @@ func resourceScalewayK8SCluster() *schema.Resource {
 				Required:    true,
 				Description: "Delete additional resources like block volumes and loadbalancers on cluster deletion",
 			},
+			"private_network_id": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				Description:  "The ID of the cluster's private network",
+				ValidateFunc: validationUUIDorUUIDWithLocality(),
+			},
```

Comment:
In resource read you set a zoned ID but user might have a raw ID in its terraform config.
Add diff supress `diffSuppressFuncLocality` to avoid plan diff